### PR TITLE
DOCS-578 - Log Query Assist edits per Search team

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Query assist edits per Search team

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-578

### CODE REVIEW

File: docs/search/query-assist.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This is a Preview release. [Register here](https://docs.google.com/forms/d/e/1FA
 
 Query Assist enhances the query-building experience in Sumo Logic by providing real-time syntax suggestions, schema prompts, and partial query predictions. These enhancements simplify the process for users at all skill levels, making it easier to write accurate and efficient queries, enabling easier discovery of relevant fields, minimizing errors, and providing intelligent assistance for query creation.
 
-<img src={useBaseUrl('img/search/get-started-search/query-assist-preview1.gif')} alt="query assist demo gif" width="700"/>
+<img src={useBaseUrl('img/search/get-started-search/query-assist-preview1.gif')} alt="query assist demo gif" width="600"/>
```
Comment (kimsauce) [inline docs/search/query-assist.md:19]:
```suggestion
<img src={useBaseUrl('img/search/get-started-search/query-assist-preview1.gif')} alt="query assist demo gif" width="700"/>
```

File: docs/search/query-assist.md [hunk 2]
```diff
@@ -36,14 +36,16 @@ Query Assist enhances the query-building experience in Sumo Logic by providing r
...
 
-* **Operator suggestions**. When typing the first letters of an operator, the system displays all matching operators related to those letters, helping you quickly find and select the appropriate one. For example, typing `st` might suggest `status`, `startsWith`, or other related operators.
-* **Value suggestions**. After typing `status=`, you will be prompted with likely values such as `success`, `error`, or other status codes based on your previous queries or log data.
+* **Operator suggestions**. When typing the first letters of an operator, the system displays all matching operators related to those letters, helping you quickly find and select the appropriate one. For example, typing `co` might suggest `count`, `count_distinct`, `count_frequent`, `compare`, `compose`, or other related operators.
+* **Value suggestions**. When you start typing a source expression (e.g. `_sourceCategory=`), you will be provided with relevant built-in metadata field options to help autocomplete your query.
+
+ For example, as you start typing , the system provides relevant metadata options and autocompletes your query. Simply press the `Tab` key to accept a suggestion.
```
Comment (kimsauce) [inline docs/search/query-assist.md:44]:
```suggestion
For example, as you start typing, the system provides relevant metadata options and autocompletes your query. Simply press the `Tab` key to accept a suggestion.  
```

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by kimsauce** [comment]
@AyanGhatak @manashar4 , have you had a chance to review your edits?

**Review by AyanGhatak — APPROVED** [review]

**Pull request merged**